frontend/app/src/router.tsx

```diff
@@ -335,6 +335,330 @@ const routes: RouteObject[] = [
           },
         ],
       },
+
+      // V1 Routes - These are explicitly listed for IDE support
+
+      {
+        path: '/v1/auth',
+        lazy: async () =>
+          import('./pages/auth/no-auth').then((res) => {
+            return {
+              loader: res.loader,
+            };
+          }),
+        children: [
+          {
+            path: '/v1/auth/login',
+            lazy: async () =>
+              import('./pages/auth/login').then((res) => {
+                return {
+                  Component: res.default,
+                };
+              }),
+          },
+          {
+            path: '/v1/auth/register',
+            lazy: async () =>
+              import('./pages/auth/register').then((res) => {
+                return {
+                  Component: res.default,
+                };
+              }),
+          },
+        ],
+      },
+      {
+        path: '/v1/onboarding/verify-email',
+        lazy: async () =>
+          import('./pages/onboarding/verify-email').then((res) => {
+            return {
+              loader: res.loader,
+              Component: res.default,
+            };
+          }),
+      },
+      {
+        path: '/v1/',
+        lazy: async () =>
+          import('./pages/authenticated').then((res) => {
+            return {
+              loader: res.loader,
+              Component: res.default,
+            };
+          }),
+        children: [
+          {
+            path: '/v1/',
+            lazy: async () => {
+              return {
+                loader: function () {
+                  return redirect('/v1/workflow-runs');
+                },
+              };
+            },
+          },
+          {
+            path: '/v1/onboarding/create-tenant',
```

Comment:
Same with the `/v1/onboarding/verify-email`, `/v1/onboarding/create-tenant`, and `/v1/onboarding/invites` routes I think, because they're not tenant-scoped -- the other routes should be ok?

Reply:
fixed these! also un-versioned `/onboarding/get-started` but let me know if that one should be versioned

Reply:
Yep, that one should be versioned, IIRC the `get-started` is the tutorial but lmk if that's not right -- we're going to want a slightly different onboarding tutorial in `v1`.

Reply:
copy that